### ISSUE TITLE
Forgot password flow?

### ISSUE THREAD

**Issue by loretta-t** [community-table-market/addison-county#29]
Are you planning to add a forgot password flow? This will be an expectation of our users. 

I'm happy to provide some references is that would be helpful, though this may already be underway.

**Comment by justindoran** [comment]
Yes, we actually have the ability to do it right now, but it's disabled in the testing environment. I'll enable it and plug it into the new way we deal with authentication flows.
